getting-started/README.md

```diff
@@ -4,22 +4,143 @@ Amber is designed to make common web-development tasks simple, and easy, maintai
 
 The goal of this overview is to introduce the general concepts in Amber and give you a quick overview of how those concepts are implemented in Amber.
 
-## Conventions Over Configuration
+## Guide Assumptions
 
-Amber provides a basic organizational structure that covers class names, filenames, database table names, and other conventions. While the conventions take some time to learn, by following the conventions Amber provides you can avoid needless configuration and make a uniform application structure that makes working with various projects simple.
+This guide is designed for beginners who want to get started with a Amber application from scratch. It does not assume that you have any prior experience with Amber. However, to get the most out of it, you need to have some prerequisites installed:
```

Comment:
This guide is designed for beginners who want to get started with an Amber application from scratch. You do not need any prior experience with the Amber Framework to use the guide.

It will require that you have some prerequisites installed: